lib/node_modules/@stdlib/fs/append-file/docs/repl.txt

```diff
@@ -0,0 +1,65 @@
+
+{{alias}}( path, data[, options], clbk )
+    Asynchronously appends data to a file.
+
+    Parameters
+    ----------
+    path: string|Buffer|integer
+        Filename or file descriptor.
+
+    data: string|Buffer
+        Data to append.
+
+    options: Object|string (optional)
+        Options. If a string, the value is the encoding.
+
+    options.encoding: string|null (optional)
+        Encoding. The encoding option is ignored if the data argument is a
```

Comment:
Why would the encoding option be ignored?

Reply:
yes, there is no need at all for this comment "The encoding option is ignored if the data argument is a buffer" in append-file

Reply:
fixed, by removing the irrelevant comment